span.go

```diff
@@ -316,6 +316,15 @@ func (s *Span) End() {
 	s.SpanData = nil
 }
 
+// ParentID returns the span's Parent ID as SpanID.
+// If Span is nil or has been ended, a zero (invalid) SpanID is returned.
+func (s *Span) ParentID() SpanID {
+	if s.SpanData == nil || s.parentID.isZero() {
```

Comment:
```suggestion
	s.mu.RLock()
	defer s.mu.RUnlock()
	if s.ended() || s.parentID.isZero() {
```

`s.SpanData` must be accessed with `s.mu` held

Reply:
Having said that, I think we should create a new `Span.parentID` field, so it can be accessed even after the span has been ended, like TraceContext. If we do that, no mutex will be required, as parent ID will be immutable.